src/vmm/src/builder.rs

```diff
@@ -5,6 +5,7 @@
 
 #[cfg(target_os = "macos")]
 use crossbeam_channel::unbounded;
+use kvm_bindings::KVM_MAX_CPUID_ENTRIES;
```

Comment:
Dropped.

Reply:
Without this commit, `cargo clippy` fails on non-`tee` builds... should we ignore?

Reply:
Are those issues being introduced by other commits in this PR?

Reply:
No, It specifically points to this code block.

Reply:
Yes, because the use of this is only in code behind a `tee` flag. Before, I don't think this was the case.. it might have been behind an `amd-sev` flag?